app/Http/Controllers/Api/BitrixImportController.php

```
@@ -182,6 +182,19 @@ private function resolveResponsibleUser(array $responsibleData, int $companyId):
         $user->status = 'active';
         $user->save();
 
+        // Create EmployeeDetails first (before attaching roles)
+        // This matches the pattern in EmployeeController and may be expected by observers/events
+        // Create a local $employee variable as attachRole or observers may expect it in scope
+        $employee = EmployeeDetails::firstOrCreate(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$employee'. (undefined)

(UnusedLocalVariable)